packages/language-server/src/plugins/svelte/SveltePlugin.ts

```diff
@@ -131,15 +172,22 @@ export class SveltePlugin
                   )
               ];
 
-        function getSveltePlugin() {
+        async function getSveltePlugin(plugins: string[] = []) {
             // Only provide our version of the svelte plugin if the user doesn't have one in
             // the workspace already. If we did it, Prettier would - for some reason - use
             // the workspace version for parsing and the extension version for printing,
             // which could crash if the contract of the parser output changed.
-            const hasPluginLoadedAlready = prettier
-                .getSupportInfo()
-                .languages.some((l) => l.name === 'svelte');
-            return hasPluginLoadedAlready ? [] : [require.resolve('prettier-plugin-svelte')];
+            return !isFallback && (await hasSveltePluginLoaded(prettier, plugins))
+                ? []
+                : [require.resolve('prettier-plugin-svelte')];
+        }
+
+        async function hasSveltePluginLoaded(p: typeof prettier, plugins: string[] = []) {
+            if (plugins.includes('prettier-plugin-svelte')) return true;
```

Comment:
It's possible that plugins have a resolved full path, but probably an edge case. When we had the problem with plugins not loading in monorepo, someone suggested in the issue using `.prettierrc.cjs` with 
```js
module.exports = {
    plugins: [require.resolve('prettier-plugin-svelte')],
}
```